rearrange/environment.py

```diff
@@ -736,6 +764,11 @@ def drop_held_object_with_snap(self) -> bool:
             if held_obj is None:
                 return False
 
+            # When dropping up an object, make it breakable.
+            self.controller.step(
+                "MakeObjectBreakable", objectId=self.held_object["objectId"]
+            )
+
```

Comment:
It doesn't look like breakable objects are ever being set to `Unbreakable`, and, given that a breakable object starts with `breakable=True`, why is this necessary?

Reply:
We do make objects unbreakable in some cases to avoid some edge cases where objects placed in their correct locations would break.

Reply:
I see. It happens in `TeleportObject` with a new
```python
makeTeleport=True
```
parameter.
https://github.com/allenai/ai2thor-rearrangement/blob/main/rearrange/environment.py#L789